src/d3d12/D3D12_Functions.cpp

```diff
@@ -497,13 +495,13 @@ void D3D12::PrepareUpdate()
 
     for (auto i = 0; i < drawData.CmdListsCount; ++i)
         IM_DELETE(drawData.CmdLists[i]);
-    delete[] drawData.CmdLists;
```

Comment:
this is wrong, it leaks memory. revert. both of these lines.

IM_DELETE does not remove all fo the data, only the inner data.

Reply:
i now understand why you did it i suppose, looking at the line right after...

you are corrupting memory, revert this as a whole, the file shouldnt be changed